alicloud/service_alicloud_ecs.go

```diff
@@ -766,3 +766,31 @@ func (s *EcsService) QueryInstanceAllDisks(id string) ([]string, error) {
 	}
 	return ids, nil
 }
+
+func (s *EcsService) DescribeLaunchTemplate(id string) (set ecs.LaunchTemplateVersionSet, err error) {
+	req := ecs.CreateDescribeLaunchTemplateVersionsRequest()
+	req.LaunchTemplateId = id
+
+	raw, err := s.client.WithEcsClient(func(ecsClient *ecs.Client) (interface{}, error) {
+		return ecsClient.DescribeLaunchTemplateVersions(req)
+	})
+	if err != nil {
+		if IsExceptedError(err, "InvalidLaunchTemplate.NotFound") {
+			err = GetNotFoundErrorFromString(fmt.Sprintf("The specified Launch Template is not found by id %s", id))
```

Comment:
The line 779 should be `err=WrapErrorf(err, NotFoundMsg, ALibabaCloudGoSdkError)`